src/adapter/src/coord/message_handler.rs

```diff
@@ -241,7 +245,7 @@ impl Coordinator {
         self.handle_command(cmd).await;
     }
 
-    #[tracing::instrument(level = "debug", skip_all)]
+    #[tracing::instrument(level = "debug", skip(self))]
```

Comment:
Two lines down the `TRACE` can be used to dump the full message content. My intention at the time was that debug could safely be turned on and wouldn't overwhelm tracing, and trace could be used sparingly if we really really needed to see the full message content. Not opposed to changing as here if there's a good reason to though.

Reply:
That totally makes sense, reverted this back to `skip_all`